allure-report-face/src/components/pane-set/styles.css

```diff
@@ -64,3 +63,7 @@
     }
   }
 }
+
+.pane + .pane {
+  box-shadow: 0 0 3px #eee;
+}
```

Comment:
This pr doesn't take a goal to completely change everything in the best way. I'll add more shadows to provide better UX in many places, but as the next separate step. For panels they looks good imho, as they split one part from another. Now maybe its not the best solution, but its better than it was. This pr fixes the most complex for the everyday work parts.